Misc/Positions_Area Positions 1 UK.txt

```diff
@@ -36,14 +36,18 @@ MAN_W_CTR:Scottish Control:128.050:MW:5:MAN:CTR:-:-:0501:0577:N053.23.30.998:W00
 SCO_CTR:Scottish Control:135.520:S:A:SCO:CTR:-:-:3601:3632:N055.37.30.606:E000.19.25.226:N059.14.08.869:W003.47.12.855:N055.52.36.972:W006.07.43.277
 SCO_E_CTR:Scottish Control:121.320:SE:E:SCO:CTR:-:-:3601:3632:N055.37.30.606:E000.19.25.226:N059.14.08.869:W003.47.12.855
 SCO_WD_CTR:Scottish Control:133.200:SWD:W:SCO:CTR:-:-:3601:3632:N055.52.36.972:W006.07.43.277
+
+SCO_N_CTR:Scottish Control:129.220:SN:W:SCO:CTR:-:-:3601:3632:N059.02.05.447:W002.55.24.968
```

Comment:
Since SN/SS/SC are going to be our event splits and not available day-to-day it would be my preference to keep them seperate, lower down - in Positions 3.

Reply:
> Since SN/SS/SC are going to be our event splits and not available day-to-day it would be my preference to keep them seperate, lower down - in Positions 3.